spec/acceptance/suites/doubleforward/01_client_tls_server_no_tls_nextserver_spec.rb

```diff
@@ -93,6 +93,7 @@ class { 'rsyslog::server':
       # Forward plain TCP to our remote log servers
       rsyslog::rule::remote { 'send_the_logs_plain_tcp':
         stream_driver => 'tcp',
+        stream_driver_permitted_peers => '',
```

Comment:
Why was this added?  stream_driver_permitted_peers does not apply unless TLS is being used.